src/proof/alethe/alethe_post_processor.cpp

```diff
@@ -1384,6 +1384,250 @@ bool AletheProofPostprocessCallback::update(Node res,
                               {},
                               *cdp);
     }
+    // ======== Trichotomy of the reals
+    // See proof_rule.h for documentation on the ARITH_TRICHOTOMY rule. This
+    // comment uses variable names as introduced there.
+    //
+    // If C = (= x c) or C = (> x c) pre-processing has to transform (>= x c)
+    // into (<= c x)
+    //
+    // ------------------------------------------------------ LA_DISEQUALITY
+    //  (VP1: (cl (or (= x c) (not (<= x c)) (not (<= c x)))))
+    // -------------------------------------------------------- OR
+    //  (VP2: (cl (= x c) (not (<= x c)) (not (<= c x))))
+    //
+    // If C = (> x c) or C = (< x c) post-processing has to be added. In these
+    // cases resolution on VP2 A B yields (not (<=x c)) or (not (<= c x)) and
+    // comp_simplify is used to transform it into C. Otherwise,
+    //
+    //  VP2   A   B
+    // ---------------- RESOLUTION
+    //  (cl C)*
+    //
+    // * the corresponding proof node is C
+    case PfRule::ARITH_TRICHOTOMY:
+    {
+      bool success = true;
+      Node equal;
+      Node lesser;
+      Node greater;
+
+      if (res.getKind() == kind::EQUAL)
```

Comment:
```suggestion
      Kind k = res.getKind();
      if (k == kind::EQUAL)
```
Also change occurrences below.